### PR TITLE
Settings

### PR DESCRIPTION
**Pull request by Sammy-F**
Completed settings with limited functions - logout still needs to be implemented.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -135,5 +153,79 @@
...
+        <item>0</item>
+        <item>-1</item>
+    </string-array>
+
+    <!-- Example settings for Data & Sync -->
+    <string name="pref_header_data_sync">Data &amp; sync</string>
+
+    <string name="pref_title_sync_frequency">Sync frequency</string>
+    <string-array name="pref_sync_frequency_titles">
+        <item>15 minutes</item>
```
Comment (MackHartley) [inline app/src/main/res/values/strings.xml]: What are these for? They look cool! Just not sure what they do

File: app/src/main/res/xml/filepaths.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<paths xmlns:android="http://schemas.android.com/apk/res/android">
+    <files-path
+        name="mFiles"
```
Comment (MackHartley) [inline app/src/main/res/xml/filepaths.xml]: Also, how does this work? I'm just curious because I've never seen this before!

File: .idea/misc.xml [hunk 3]
```diff
@@ -1,8 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (MackHartley) [inline .idea/misc.xml]: Might need to be added to gitignore?

File: .idea/modules.xml [hunk 4]
```diff
@@ -2,7 +2,8 @@
 <project version="4">
```
Comment (MackHartley) [inline .idea/modules.xml]: Might need to be added to gitignore?

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (Sammy-F) [reply app/src/main/res/values/strings.xml]: Sorry, I think these are actually Strings that were added automatically when I created the Settings Activity. I'll remove them now.

File: app/src/main/res/xml/filepaths.xml (hunk 2 above)
Comment (Sammy-F) [reply app/src/main/res/xml/filepaths.xml]: This is the filepath I had to define for passing information from the app to an email app via the Content Provider. We don't really actively use it anywhere - it's more a background thing.

**Review by MackHartley — CHANGES_REQUESTED** [review]

**Review by MackHartley — APPROVED** [review]

**Comment by MackHartley** [comment]
Sorry about the wait, gonna merge this now, it all looks great!

**Pull request merged**